bloomstack_core/public/js/sales_order_list.js

```diff
@@ -1,274 +0,0 @@
-								let created_orders = r.message.filter(order => order.created === true);
-								for (let order of created_orders) {
-									let delivery_notes = order.delivery_notes
-										.map(delivery_note => frappe.utils.get_form_link("Delivery Note", delivery_note, true))
-										.join(", ");
-
-									created_order_message += `<li><strong>${order.customer}</strong> (${order.sales_order}): ${delivery_notes}</li>`;
-								}
-
-								if (created_order_message) {
-									message += `The following Delivery Note were created:<br><br><ul>${created_order_message}</ul>`;
-								}
-
-								// loop through each existing order and render linked Pick Lists
-								let existing_order_message = "";
-								let existing_orders = r.message.filter(order => order.created === false);
-								for (let order of existing_orders) {
-									let delivery_notes = order.delivery_notes
-										.map(delivery_note => frappe.utils.get_form_link("Delivery Note", delivery_note, true))
-										.join(", ");
-
-									existing_order_message += `<li><strong>${order.customer}</strong> (${order.sales_order}): ${delivery_notes || "No available items to pick"}</li>`;
-								}
-
-								if (existing_order_message) {
-									message += `<br>The following orders either have existing Delivery Note:<br><br><ul>${existing_order_message}</ul>`;
-								}
-
-								frappe.msgprint(__(message));
-
-								// if validation messages are found, append at the bottom of our message
-								if (r._server_messages) {
-									let server_messages = JSON.parse(r._server_messages);
-									for (let server_message of server_messages) {
-										frappe.msgprint(__(JSON.parse(server_message).message));
-									}
-									// delete server messages to avoid Frappe eating up our msgprint
-									delete r._server_messages;
-								}
-
-								doclist.refresh();
-							}
-						}
-					});
-				})
-		};
-	};
-
-	doclist.page.add_actions_menu_item(__('Create Pick Lists'), create_pick_list_action, false);
-	doclist.page.add_actions_menu_item(__('Create Sales Invoices'), create_sales_invoice_action, false);
-	doclist.page.add_actions_menu_item(__('Create Delivery Note'), create_delivery_note_action, false);
-
-	const send_email_action = () => {
-		const selected_docs = doclist.get_checked_items();
-		const doctype = doclist.doctype;
-		if (selected_docs.length > 0) {
-			let title = selected_docs[0].title;
-			for (let doc of selected_docs) {
-				if (doc.docstatus !== 1) {
-					frappe.throw(__("Cannot Email Draft or cancelled documents"));
-				}
-				if (doc.title !== title) {
-					frappe.throw(__("Select only one customer's sales orders"))
-				}
-			};
-			frappe.call({
-				method: "bloomstack_core.utils.get_contact",
-				args: { "doctype": doctype, "name": selected_docs[0].name, "contact_field": "customer" },
-				callback: function (r) {
-					frappe.call({
-						method: "bloomstack_core.utils.get_document_links",
```

Comment:
@nmimsnikhil are you taking care of this function. if not calling from any other function then remove this function from utils.js

Reply:
@vishdha i have checked that this functions are not calling from anywhere